install/tools/ipa-compat-manage

```diff
@@ -100,7 +100,10 @@ def main():
         if dirman_password is None:
             sys.exit("Directory Manager password required")
 
-    api.bootstrap(context='cli', in_server=True, debug=options.debug)
+    api.bootstrap(context='cli',
+                  in_server=True,
+                  debug=options.debug,
+                  confdir=paths.ETC_IPA)
```

Comment:
You're still required to enter Directory Manager password which is why I would expect the script not to be influenced by an environment variable.